source/core/GenericFunctions.cpp

```diff
@@ -2437,6 +2437,84 @@ VIREO_FUNCTION_SIGNATURE4(ConvertEnum, StaticType, void, StaticType, void)
     return _NextInstruction();
 }
 //------------------------------------------------------------
+// arguments: output cluster, then variable number of inputs that can be null, error cluster or a 1D array of error clusters
+struct MergeErrorsParamBlock : public VarArgInstruction
+{
+    _ParamDef(ErrorCluster, ErrorClusterOut);
+    _ParamImmediateDef(StaticTypeAndData, ErrorClusterInputs[1]);
+    NEXT_INSTRUCTION_METHODV()
+};
+
+static Boolean UpdateErrorWhenErrorOrWarningFound(ErrorCluster* errorClusterIn, ErrorCluster* errorClusterOut, Boolean findErrors) {
+    Boolean errorFound = errorClusterIn->status && findErrors;
+    Boolean warningFound = !errorClusterIn->status && errorClusterIn->code != 0 && !findErrors;
```

Comment:
Agree with suggestion, but I'd prefer hasError() and hasWarning(), because isError() could be misread as if it's querying the type rather than the data.